### PR TITLE
Initial database schema

### PR DESCRIPTION
**Pull request by sbelov**

### CODE REVIEW

File: awesomeagile-dao/pom.xml [hunk 1]
```diff
@@ -23,5 +23,27 @@
       <groupId>com.zaxxer</groupId>
       <artifactId>HikariCP</artifactId>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
```
Comment (mwarren) [inline awesomeagile-dao/pom.xml]: We probably want test scope on this.

File: awesomeagile-dao/pom.xml [hunk 2]
```diff
@@ -23,5 +23,27 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.awesomeagile</groupId>
+      <artifactId>awesomeagile-model</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>compile</scope>
```
Comment (mwarren) [inline awesomeagile-dao/pom.xml]: Duplicate dependency?

File: awesomeagile-dao/pom.xml (hunk 1 above)
Comment (sbelov) [inline awesomeagile-dao/pom.xml]: addressed

File: awesomeagile-dao/pom.xml (hunk 2 above)
Comment (sbelov) [inline awesomeagile-dao/pom.xml]: addressed

**Pull request merged**